src/Whoops/Util/TemplateHelper.php

```diff
@@ -72,6 +72,39 @@ public function escapeButPreserveUris($raw)
         );
     }
 
+    /**
+     * Makes sure that the given string breaks on the delimiter.
+     *
+     * @param  string $delimiter
+     * @param  string $s
+     * @return string
+     */
+    public function breakOnDelimiter($delimiter, $s)
```

Comment:
No it's not the same. By wrapping the parts with a div, browsers will not break the text inside a part, but put the div on the next line. Also, browsers put a dash after soft hypen.
Compare the pictures in the pull request description.